api/v1alpha1/artifact.go

```diff
@@ -42,6 +42,11 @@ type Artifact struct {
 	// update of this artifact.
 	// +required
 	LastUpdateTime metav1.Time `json:"lastUpdateTime,omitempty"`
+
+	// IgnoredPatterns returns the patterns that were used to shape the artifact.
+	// Populated by (*Storage).Archive()
+	// +optional
+	IgnoredPatterns []string `json:"ignoredPatterns,omitempty"`
```

Comment:
This should be on the spec not status, and only on the GitRepository object.

Reply:
I'm still adding that; this is more for *reporting* what was ignored to generate the tarball. I figured it'd be useful, but I can remove it if you don't like it.

Reply:
I see no point in duplicating a field, please remove it. Also this doesn't make sense on HelmChart/Repo.

Reply:
Also please always run `make test` before a commit, that command will keep the CRD yamls in docs in sync with the code.

Reply:
Ok, will do.

Reply:
removed; I've also added the basic ignore functionality